hoodie-client/src/main/java/com/uber/hoodie/HoodieWriteClient.java

```diff
@@ -822,6 +818,39 @@ public void startCommitWithTime(String commitTime) {
             new HoodieInstant(true, commitActionType, commitTime));
     }
 
+    /**
+     * Performs a compaction operation on a MergeOnRead type of dataset. Depending on the CompactionStrategy,
```

Comment:
reword comment to remove reference to merge-on-read . Like to leave this file free of storage types.

Reply:
done.